<?xml version="1.0" encoding="UTF-8"?>
<table id="PMC7481248/table_4" xmlns="http://www.w3.org/1999/xhtml">
 <caption>
  <label>Table 4</label>
  <p>Studies based on machine learning for PD biomarker identification.</p>
 </caption>
 <thead>
  <tr>
   <th colspan="1" rowspan="1" id="0-0-.">Study</th>
   <th colspan="1" rowspan="1" id="0-1-.">Genetic data</th>
   <th colspan="1" rowspan="1" id="0-2-.">Participants</th>
   <th colspan="1" rowspan="1" id="0-3-.">Model</th>
   <th colspan="1" rowspan="1" id="0-4-.">Validation</th>
   <th colspan="1" rowspan="1" id="0-5-.">Biomarkers</th>
  </tr>
 </thead>
 <tbody>
  <tr>
   <td>Scherzer et al. 2007 
    <sup>
     <a ref-type="bibr" rid="CR54">54</a>
    </sup>
   </td>
   <td>Microarrays from whole blood samples</td>
   <td>50 PDs and 55 HCs</td>
   <td>Supervised classification model</td>
   <td>Validation sample set</td>
   <td>8 genes: VDR, HIP2, CLTB, FPRL2, CA12, CEACAM4, ACRV1, and UTX</td>
  </tr>
  <tr>
   <td>Molochnikov et al. 2012 
    <sup>
     <a ref-type="bibr" rid="CR55">55</a>
    </sup>
   </td>
   <td>Microarrays from blood samples</td>
   <td>38 de novo PDs, 24 early stage PDs (within 1st-year medication), 30 advanced PDs, 29 ADs and 64 HCs</td>
   <td>Stepwise multivariate logistic regression model</td>
   <td>Validation sample set</td>
   <td>5 genes: HIP2, ALDH1A1, PSMC4, HSPA8 and EGLN1</td>
  </tr>
  <tr>
   <td>Potashkin et al. 2012 
    <sup>
     <a ref-type="bibr" rid="CR56">56</a>
    </sup>
   </td>
   <td>Splice variant-specific microarrays</td>
   <td>51 PDs, 34 APD (17 MSA and 17 PSP), 39 HCs</td>
   <td>KNN</td>
   <td>qPCR</td>
   <td>13 genes: C5ORF4, WLS, MACF1, PRG3, EFTUD2, PKM2, SLC14A1-S, SLC14A1-L, MPP1, COPZ1, ZNF160, MAP4K1 and ZNF134</td>
  </tr>
  <tr>
   <td>Santiago et al. 2013 
    <sup>
     <a ref-type="bibr" rid="CR59">59</a>
    </sup>
   </td>
   <td>Gene expression data from blood samples</td>
   <td>50 PDs and 46 HCs</td>
   <td>Stepwise multivariate LDA</td>
   <td>None</td>
   <td>7 out of 13 genes in previous study 
    <sup>
     <a ref-type="bibr" rid="CR56">56</a>
    </sup>: C5ORF4, COPZ1, MACF1, WLS, PRG3, ZNF160 and EFTUD2 
   </td>
  </tr>
  <tr>
   <td>Karlsson et al. 2013 
    <sup>
     <a ref-type="bibr" rid="CR60">60</a>
    </sup>
   </td>
   <td>Microarrays from blood samples</td>
   <td>79 PDs and 75 HCs</td>
   <td>CPLS</td>
   <td>None</td>
   <td>6 genes: LRPPRC, BCL2, SRSF8, HSPA8, UBE2K, EGLN1</td>
  </tr>
  <tr>
   <td>Calligaris et al. 2015 
    <sup>
     <a ref-type="bibr" rid="CR57">57</a>
    </sup>
   </td>
   <td>Microarrays from blood samples</td>
   <td>52 PDs and 32 HCs</td>
   <td>PLS-DA and LDA (a Bayesian classification method)</td>
   <td>RT-qPCR</td>
   <td>54 genes</td>
  </tr>
  <tr>
   <td>Shamir et al. 2017 
    <sup>
     <a ref-type="bibr" rid="CR61">61</a>
    </sup>
   </td>
   <td>Microarrays from blood samples</td>
   <td>205 PDs, 233 HCs and 48 other neurodegenerative diseases</td>
   <td>SVM</td>
   <td>None</td>
   <td>A gene signature of 87 gene (64 upregulated and 23 downregulated genes between PD and HC)</td>
  </tr>
 </tbody>
 <tfoot>
  <p>
   <i>AD</i> Alzheimer’s disease, 
   <i>APD</i> Atypical parkinsonian disorders, 
   <i>CPLS</i> canonical partial least squares, 
   <i>HC</i> Healthy control, 
   <i>KNN</i> K-nearest neighbor classification, 
   <i>LDA</i> linear discriminant regression (a Bayesian classification method), 
   <i>MSA</i> multiple system atrophy, 
   <i>PD</i> Parkinson’s disease, 
   <i>PSP</i> progressive supra-nuclear palsy, 
   <i>qPCR</i> quantitative polymerase chain reaction, 
   <i>SN</i> substantia nigra, 
   <i>SVM</i> support vector machine. 
  </p>
 </tfoot>
</table>
